Properly link libfcgi++ against libfcgi

We cannot directly use -lfcgi to link libfcgi++ against libfcgi,
because libfcgi is not installed at this time. Instead, we should rely
on libtool doing the right thing, by specifying libfcgi.la in
libfcgi++ LIBADD variable.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Index: b/libfcgi/Makefile.am
===================================================================
--- a/libfcgi/Makefile.am
+++ b/libfcgi/Makefile.am
@@ -23,5 +23,6 @@
                        $(INCLUDEDIR)/fcgio.h  \
                        fcgio.cpp
 libfcgi___la_CFLAGS  = @PTHREAD_CFLAGS@
-libfcgi___la_LDFLAGS = -lfcgi -rpath @libdir@
+libfcgi___la_LIBADD  = libfcgi.la
+libfcgi___la_LDFLAGS = -rpath @libdir@
